start

```diff
@@ -419,14 +434,25 @@ function init_soroban_rpc() {
     echo "soroban rpc: already initialized"
     return 0
   fi
+
+  SOROBAN_RPC_CAPTIVE_CORE_CFG=$COREHOME/etc/soroban-rpc-captive-core.cfg
+  cp $COREHOME/etc/stellar-captive-core.cfg $SOROBAN_RPC_CAPTIVE_CORE_CFG 2>/dev/null || true
```

Comment:
Why do we allow the copy to fail with the `|| true`?

Reply:
obsolete now, code was removed as part of adding captive core .cfg for each network/service instead, https://github.com/stellar/quickstart/pull/414/commits/f6e66c3d67b23383a036ce3db183b4b1f9ef910c